src/cdomains/intDomain.ml

```diff
@@ -1296,15 +1277,15 @@ struct
     | `Excluded (s,l) when S.is_empty s -> "Unknown int" ^ short_size l
     (* Prepend the exclusion sets with something: *)
     | `Excluded (s,l) -> "Not " ^ S.show s ^ short_size l
+
+  include Std (struct type nonrec t = t let name = name let top_of = top_of let bot_of = bot_of let show = show let equal = equal end)
+  (* FIXME: poly compare? *)
```

Comment:
`Std` does not contain a `compare` function anymore, so one can remove the comment.

Reply:
Felt like I had already done it, but apparently that's in #542. Did it here as well.